plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceRuntimeInfo.java

```diff
@@ -299,9 +315,10 @@ public String projectsRoot() {
             String externalPort = portEntry.getValue().get(0).getHostPort();
             servers.put(portProtocol, new ServerImpl(null,
                                                      null,
-                                                     host + ":" + externalPort,
+                                                     externalHostame + ":" + externalPort,
                                                      null,
-                                                     null));
+                                                     externalPort,
+                                                     new ServerPropertiesImpl(null, internalHostname + ":" + externalPort, null)));
```

Comment:
After fixup `externalHostname`, `internalHostname` and `externalPorts` can never be `null`.